src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx

```diff
@@ -25,32 +28,69 @@ export default function CSVFileImport({ url, title }: CSVFileImportProps) {
 
   const uploadFile = async () => {
     if (!file) {
-      console.error("No file selected");
+      showAlert({ severity: "error", message: "No file selected" });
       return;
     }
 
     console.log("uploadFile to", url);
-    // Get the presigned URL
-    const response = await axios({
-      method: "GET",
-      url,
-      params: {
-        name: encodeURIComponent(file.name),
-      },
-    });
 
-    console.log("File to upload: ", file.name);
-    console.log("Uploading to: ", response.data.url);
-    const result = await fetch(response.data.url, {
-      method: "PUT",
-      body: file,
-      headers: {
-        "Content-Type": file.type,
-      },
-    });
-    console.log("Result: ", result);
-    setFile(undefined);
+    const authorization_token = localStorage.getItem("authorization_token");
+    const headers: any = {};
+    if (authorization_token) {
+      headers.Authorization = `Basic ${authorization_token}`;
+    }
+    try {
+      const response = await axios({
+        method: "GET",
+        url,
+        params: {
+          name: encodeURIComponent(file.name),
+        },
+        headers,
+      });
+
+      const result = await fetch(response.data.url, {
+        method: "PUT",
+        body: file,
+        headers: {
+          "Content-Type": file.type,
+        },
+      });
+      console.log("Result: ", result);
+      setFile(undefined);
+      showAlert({
+        severity: "success",
+        message: "File uploaded successfully!",
+      });
+    } catch (error) {
+      if (axios.isAxiosError(error)) {
```

Comment:
Local error handling is implemented which is fine, but accordingly to the spec global handling should have been implemented in `index.tsx`.
Please find a way to implement global error handler for Axios at https://axios-http.com/docs/interceptors